jax/experimental/jax2tf/examples/tflite/mnist/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST classification
+
+This directory contains the code necessary to train a simple convolution network on the MNIST dataset, and subsequently convert it using jax2tf into a model that can be converted to tflite format.
+
+This example is based on the [TensorFlow Lite demo of building a handwritten digit classifier app with TensorFlow Lite](https://developer.android.com/codelabs/digit-classifier-tflite). The model code is taken from [FLAX Linen Mnist Example](https://github.com/google/flax/tree/master/linen_examples/mnist).
+
+## Requirements
+
+* TensorFlow dataset mnist will be downloaded and prepared automatically, if necessary
+
+## Training the model
+
+You can train and export the model to tflite format yourself running
+```bash
+$ python mnist.py
+```
```

Comment:
Suggestion (I may be wrong about the code style you use here though but do add a blank line before the code block and reformat some words):

@qiuminxu LMKWYT of this (hope I got the gist right - essentially, `mnist.py` performs the entire thing end-to-end including quantization, right? I hope I got it right 😄 Let me know if I'm incorrect)

```suggestion
To train the model in JAX with the Flax library, convert it into a TensorFlow SavedModel, and export it into the TF Lite format, run the following command:

```shell
python mnist.py
```
```

Reply:
@qiuminxu 

I think the rendering of triple backticks in my previous suggestion here caused a set of them to "fall off" - and I didn't spot it:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/19637339/101399698-bc13e580-38c7-11eb-815c-0bbffe2cb4d5.png">

As a result, there's a small rendering issue on the changed Markdown:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/19637339/101399685-b61e0480-38c7-11eb-9e5c-6618f321bf48.png">

Oops!

Do you mind adding the backticks back to close the code block?

    ```shell
    python mnist.py
    ```

🙏 Thanks!!